frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-card border="md" @click="console.log()">
+    <v-row class="ma-0">
+      <v-col cols="7" class="pa-0 border-e-md">
+        <v-card-title class="text-h6 text-center">
```

Comment:
It looks like some of these names truncate. Can we make this wrap instead so they never lose that information?

Reply:
Seems like a lot of that is coming from `<v-card-title>`, so we _may_ want to use another element, if we don't feel like overriding several classes from Vuetify.

Reply:
I've set v-card-title to wrap text by default in the plugin options, that's the only source